wire-compiler/src/main/java/com/squareup/wire/WireCompiler.java

```diff
@@ -111,10 +118,14 @@
   final boolean namedFilesOnly;
   final boolean emitAndroid;
   final boolean emitCompact;
+  final ImmutableMap<ProtoType, TypeName> protoTypeToJavaClassName;
+  final ImmutableMap<ProtoType, AdapterEntry> protoTypeToAdapter;
```

Comment:
nit: protoTypeToAdapter –> protoTypeToAdapterEntry ? (otherwise the name is a lie?)